### PR TITLE
Keep multi selection on table refresh

### PR DESCRIPTION
**Pull request by snichme**
The code around multi selection has been refactored into a separate "module" so all the variables and functions dont mix with the other code, this allows the handler (multiActionHandler) to only expose a small set of methods to work on for the multi select feature.

Part from the refactoring, the main change implemented is that we keep a local hash to keep track of which checkbox was checked so we can marked them as checked again when the table refreshes see method `isSelected`

### CODE REVIEW

**Comment by spuun** [comment]
Ouch! Just realized that this feature is partly broken. If you select a few rows, then wait until table is updated (5 seconds) selection will be gone.

File: static/js/queues.js [hunk 1]
```diff
@@ -4,6 +4,84 @@ import * as Dom from './dom.js'
...
+  }
+
+  const id = (vhost, name) => `${vhost}/${name}`
+
+  const onSelectionChange = () => {
+    const checked = document.querySelectorAll("input[data-name]:checked")
+    selectedQueues = {}
+    checked.forEach(el => {
+      const data = el.dataset
+      selectedQueues[id(data.vhost, data.name)] = true
```
Comment (spuun) [inline static/js/queues.js:51]: `data.vhost` is uri encoded, so `/` is `%2F` here but not down in `isSelected`

**Review by spuun — COMMENTED** [review]
I think the bug this will fix was introduced by me when i fixed the "edit entities" feature. I'm refactoring tables scripts right now, so I'll make sure not to break it again (preferably checkboxes won't be recreated every refresh, and then this change won't be needed anymore).

**Comment by snichme** [comment]
Ok sounds good. Closing this PR

**Pull request closed without merging**